### PR TITLE
Add the function of select columns

### PR DESCRIPTION
**Pull request by dannyball710**

### CODE REVIEW

File: src/queryBuilder.ts [hunk 1]
```diff
@@ -70,7 +73,11 @@ export function GenerateQuery<T extends object>(
 	const bindings: unknown[] = [];
 	switch (type) {
 		case QueryType.SELECT: {
-			query = `SELECT * FROM \`${tableName}\``;
+			let columns = "*";
+			if (options.columns && options.columns.length > 0) {
+				columns = options.columns.map((x) => `\`${String(x)}\``).join(", ");
```
Comment (Skye-31) [inline src/queryBuilder.ts:78]: Can we use `"` (double quotes) for this instead, as is the standard for column names?

File: test/querybuilder.test.js [hunk 2]
```diff
@@ -112,6 +112,12 @@ describe("Query Builder", () => {
 				expect(statement.bindings[0]).to.equal(1);
 				expect(statement.bindings[1]).to.equal("test");
 			});
+			it("should generate a query with a select columns", () => {
+				const statement = GenerateQuery(QueryType.SELECT, "test", {
+					columns: ["id", "name"]
+				});
+				expect(statement.query).to.equal("SELECT `id`, `name` FROM `test`");
```
Comment (Skye-31) [inline test/querybuilder.test.js:119]: Likewise

**Review by Skye-31 — CHANGES_REQUESTED** [review]
Thanks for the PR!

A minor change to match the SQLite standard would be wonderful. Please also run the `changeset` and `prettify` scripts 🙂

**Comment by dannyball710** [comment]
Understood, let me fix it.

File: src/model.ts [hunk 3]
```diff
@@ -224,12 +224,17 @@ export class Model<T extends Record<string, ModelColumn>> {
 	 * @returns Returns the first row that matches the where clause, or null if no rows match.
 	 */
 	public async First(
-		options: Pick<GenerateQueryOptions<InferFromColumns<T>>, "where">
+		options: Pick<
+			GenerateQueryOptions<InferFromColumns<T>>,
+			"where" | "columns"
+		>
 	): Promise<InferFromColumns<T> | null> {
```
Comment (Skye-31) [inline src/model.ts:231]: One thing I've realised is that when the user is specifying `columns`, the return type of this method is going to be wrong - we'd only want to return the type for the columns which are being returned

There's not any particularly nice way to handle this though 🤔

**Pull request closed without merging**